charts/nri-bundle/values.yaml

```diff
@@ -11,9 +15,11 @@ nri-metadata-injection:
   enabled: true
 
 kube-state-metrics:
-  # kube-state-metrics.enabled -- Install the [`kube-state-metrics` chart from the stable helm charts repository](https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics)
-  # This is mandatory if `infrastructure.enabled` is set to `true` and the user does not provide its own instance of KSM version >=1.8 and <=2.0
-  enabled: false
+  # kube-state-metrics.enabled -- Install the [`kube-state-metrics` chart](https://kubernetes.github.io/kube-state-metrics/). kube-state-metrics is a mandatory dependency of the integration.
+  # If kube-state-metrics >= v1.8.0 <= 1.9.8 is already installed in your cluster, you can set this to `false` and adjust `newrelic-infrastructure.ksm.config.selector` accordingly to match the installed KSM label.
+  enabled: true
+  # -- Use the following name and app label for NR-bundled KSM.
+  nameOverride: nrk8s-kube-state-metrics
```

Comment:
How about to add a new label like `newrelic.io/ksm-instance: true` just to avoid changing an existing label that it might be already used ?

Reply:
I think the purpose of breaking the label is two-fold. The first one is to prevent our integration from autoselecting other KSM when ours is installed. The second one I see is to "claim ownership" of this KSM. After breaking this label, we can start tweaking it more heavily than before: Removing `collectors`, bumping its major without bumping ours, etc. I think this is desirable for us and will cause us to move faster with it, but would not work if we don't get rid of the original label.